### PR TITLE
feat(Testimonial): Add new Testimonial Component

### PR DESCRIPTION
**Pull request by cstatro**
### Overview

This PR adds a new  `Testimonial` component to gamut that utilizes the `FloatingCard` component.  The new component has updated styles  and support for light/dark  color modes. It has two variants horizontal/vertical but  any implementation will go to vertical on mobile viewports.  

image

### PR Checklist

- [x] Related to designs:
- [x] Related to JIRA ticket: GROW-2490
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.2-alpha.8c731f.0<br/>@codecademy/gamut-illustrations@0.17.16-alpha.8c731f.0<br/>@codecademy/gamut-kit@0.5.22-alpha.8c731f.0<br/>@codecademy/gamut-labs@19.19.8-alpha.8c731f.0<br/>@codecademy/gamut-patterns@0.5.4-alpha.8c731f.0<br/>@codecademy/gamut-styles@11.7.2-alpha.8c731f.0<br/>@codecademy/gamut-tests@2.4.19-alpha.8c731f.0<br/>@codecademy/gamut@35.14.1-alpha.8c731f.0<br/>@codecademy/konamimojisplosion@0.2.50-alpha.8c731f.0<br/>@codecademy/markdown-overrides@0.5.62-alpha.8c731f.0<br/>@codecademy/styleguide@42.7.8-alpha.8c731f.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.2-alpha.3a0371.0<br/>@codecademy/gamut-illustrations@0.17.16-alpha.3a0371.0<br/>@codecademy/gamut-kit@0.5.22-alpha.3a0371.0<br/>@codecademy/gamut-labs@19.19.8-alpha.3a0371.0<br/>@codecademy/gamut-patterns@0.5.4-alpha.3a0371.0<br/>@codecademy/gamut-styles@11.7.2-alpha.3a0371.0<br/>@codecademy/gamut-tests@2.4.19-alpha.3a0371.0<br/>@codecademy/gamut@35.14.1-alpha.3a0371.0<br/>@codecademy/konamimojisplosion@0.2.50-alpha.3a0371.0<br/>@codecademy/markdown-overrides@0.5.62-alpha.3a0371.0<br/>@codecademy/styleguide@42.7.8-alpha.3a0371.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.2-alpha.aa5cfb.0<br/>@codecademy/gamut-illustrations@0.17.16-alpha.aa5cfb.0<br/>@codecademy/gamut-kit@0.5.22-alpha.aa5cfb.0<br/>@codecademy/gamut-labs@19.19.8-alpha.aa5cfb.0<br/>@codecademy/gamut-patterns@0.5.4-alpha.aa5cfb.0<br/>@codecademy/gamut-styles@11.7.2-alpha.aa5cfb.0<br/>@codecademy/gamut-tests@2.4.19-alpha.aa5cfb.0<br/>@codecademy/gamut@35.14.1-alpha.aa5cfb.0<br/>@codecademy/konamimojisplosion@0.2.50-alpha.aa5cfb.0<br/>@codecademy/markdown-overrides@0.5.62-alpha.aa5cfb.0<br/>@codecademy/styleguide@42.7.8-alpha.aa5cfb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6111c034148ef569c9419619--gamut-preview.netlify.app>https://6111c034148ef569c9419619--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6111c034148ef569c9419619>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.2-alpha.a237f7.0<br/>@codecademy/gamut-illustrations@0.17.16-alpha.a237f7.0<br/>@codecademy/gamut-kit@0.5.22-alpha.a237f7.0<br/>@codecademy/gamut-labs@19.19.8-alpha.a237f7.0<br/>@codecademy/gamut-patterns@0.5.4-alpha.a237f7.0<br/>@codecademy/gamut-styles@11.7.2-alpha.a237f7.0<br/>@codecademy/gamut-tests@2.4.19-alpha.a237f7.0<br/>@codecademy/gamut@35.14.1-alpha.a237f7.0<br/>@codecademy/konamimojisplosion@0.2.50-alpha.a237f7.0<br/>@codecademy/markdown-overrides@0.5.62-alpha.a237f7.0<br/>@codecademy/styleguide@42.7.8-alpha.a237f7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61128bed1612e12fc66f27f8--gamut-preview.netlify.app>https://61128bed1612e12fc66f27f8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61128bed1612e12fc66f27f8>Deploy Logs</a></p>

File: packages/gamut-labs/src/assets/navyQuotes.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="109" height="74" xmlns="http://www.w3.org/2000/svg">
```
Comment (borisonr) [inline packages/gamut-labs/src/assets/navyQuotes.svg:1]: this looks like a different svg than what tamar used in figma...

File: packages/gamut-labs/src/Testimonial/index.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+            src={imageUrl}
+            alt="testimonial"
+          />
+        )}
+        <Box
+          my={{ _: 'auto', md: isVerticleLayout && !hidePhoto ? 'auto' : 0 }}
+          mr={32}
+          gridArea={!hidePhoto ? 'byline' : 'avatar'}
+        >
+          <Text variant="p-small" as="p" fontFamily="accent">
```
Comment (borisonr) [inline packages/gamut-labs/src/Testimonial/index.tsx:129]: looks like tamar wants the name underlined?

**Review by borisonr — COMMENTED** [review]
looks so great! just a couple qs re figma discrepancies

File: packages/gamut-labs/src/assets/navyQuotes.svg (hunk 1 above)
Comment (BandanaKM) [reply packages/gamut-labs/src/assets/navyQuotes.svg:1]: had a similar thought here

**Comment by BandanaKM** [comment]
this looks wonderful, @cstatro and works great! only question is about the svg. 

another super small nit on the stories part. in the markdown, it looked correct but i wonder why we have an extra new line under hidden picture?

**Review by BandanaKM — APPROVED** [review]
generally looks great, would be curious about a review from the web platform team so wee if there are any other small things they would optimize

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.2-alpha.26a719.0<br/>@codecademy/gamut-illustrations@0.17.16-alpha.26a719.0<br/>@codecademy/gamut-kit@0.5.22-alpha.26a719.0<br/>@codecademy/gamut-labs@19.19.8-alpha.26a719.0<br/>@codecademy/gamut-patterns@0.5.4-alpha.26a719.0<br/>@codecademy/gamut-styles@11.7.2-alpha.26a719.0<br/>@codecademy/gamut-tests@2.4.19-alpha.26a719.0<br/>@codecademy/gamut@35.14.1-alpha.26a719.0<br/>@codecademy/konamimojisplosion@0.2.50-alpha.26a719.0<br/>@codecademy/markdown-overrides@0.5.62-alpha.26a719.0<br/>@codecademy/styleguide@42.7.8-alpha.26a719.0<br/></pre>

File: packages/gamut-labs/src/assets/navyQuotes.svg (hunk 1 above)
Comment (cstatro) [reply packages/gamut-labs/src/assets/navyQuotes.svg]: yep, good catch just changed it

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6112de0c5b77c67a09000c51--gamut-preview.netlify.app>https://6112de0c5b77c67a09000c51--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6112de0c5b77c67a09000c51>Deploy Logs</a></p>

**Review by borisonr — APPROVED** [review]

**Comment by cstatro** [comment]
@BandanaKM  so I spent a few minutes looking at this and no idea why this header just has extra space too it. Happy to try any suggestions anyone might have, but don't feel like it's worth digging deeper on this.

**Comment by cstatro** [comment]
@BandanaKM  just kidding I got fixated and resolved it lol. Had to set display on column to block
image

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.2-alpha.88b818.0<br/>@codecademy/gamut-illustrations@0.17.16-alpha.88b818.0<br/>@codecademy/gamut-kit@0.5.22-alpha.88b818.0<br/>@codecademy/gamut-labs@19.19.8-alpha.88b818.0<br/>@codecademy/gamut-patterns@0.5.4-alpha.88b818.0<br/>@codecademy/gamut-styles@11.7.2-alpha.88b818.0<br/>@codecademy/gamut-tests@2.4.19-alpha.88b818.0<br/>@codecademy/gamut@35.14.1-alpha.88b818.0<br/>@codecademy/konamimojisplosion@0.2.50-alpha.88b818.0<br/>@codecademy/markdown-overrides@0.5.62-alpha.88b818.0<br/>@codecademy/styleguide@42.7.8-alpha.88b818.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6112e76478ae289a1f5d4a1e--gamut-preview.netlify.app>https://6112e76478ae289a1f5d4a1e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6112e76478ae289a1f5d4a1e>Deploy Logs</a></p>

File: packages/gamut-labs/src/Testimonial/__tests__/Testimonial-test.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+describe('Testimonial', () => {
+  const defaultProps: TestimonialProps = {
+    firstName: 'Christopher',
+    lastName: 'Moltisanti',
+    imageUrl: 'okthen',
+    href: 'bloip',
+    location: 'New Jersey',
+    occupation: 'Waste Management',
+    quote:
+      "Like everybody starts out somewheres. and they do something, something gets done to them and it changes their life. That's called an arc. Where's my arc?",
```
Comment (dreamwasp) [inline packages/gamut-labs/src/Testimonial/__tests__/Testimonial-test.tsx:14]: 🙌

File: packages/gamut-labs/src/Testimonial/index.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+          _: gridLayouts.vertical,
+          md: gridLayouts.horizontal,
+        },
+      },
+      vertical: {
+        gridTemplateAreas: gridLayouts.vertical,
+      },
+    },
+  })
+);
```
Comment (dreamwasp) [inline packages/gamut-labs/src/Testimonial/index.tsx:56]: 🔥🔥🔥

**Review by dreamwasp — APPROVED** [review]
look great, awesome work!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.4-alpha.72c1c5.0<br/>@codecademy/gamut-illustrations@0.17.18-alpha.72c1c5.0<br/>@codecademy/gamut-kit@0.5.31-alpha.72c1c5.0<br/>@codecademy/gamut-labs@19.20.6-alpha.72c1c5.0<br/>@codecademy/gamut-patterns@0.5.6-alpha.72c1c5.0<br/>@codecademy/gamut-styles@11.8.1-alpha.72c1c5.0<br/>@codecademy/gamut-tests@2.4.21-alpha.72c1c5.0<br/>@codecademy/gamut@36.4.1-alpha.72c1c5.0<br/>@codecademy/konamimojisplosion@0.2.52-alpha.72c1c5.0<br/>@codecademy/markdown-overrides@0.5.64-alpha.72c1c5.0<br/>@codecademy/styleguide@43.6.1-alpha.72c1c5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6116d155dad4bd092be6b7e0--gamut-preview.netlify.app>https://6116d155dad4bd092be6b7e0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6116d155dad4bd092be6b7e0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.4-alpha.694fd6.0<br/>@codecademy/gamut-illustrations@0.17.18-alpha.694fd6.0<br/>@codecademy/gamut-kit@0.5.32-alpha.694fd6.0<br/>@codecademy/gamut-labs@19.20.7-alpha.694fd6.0<br/>@codecademy/gamut-patterns@0.5.6-alpha.694fd6.0<br/>@codecademy/gamut-styles@11.8.1-alpha.694fd6.0<br/>@codecademy/gamut-tests@2.4.21-alpha.694fd6.0<br/>@codecademy/gamut@36.4.2-alpha.694fd6.0<br/>@codecademy/konamimojisplosion@0.2.52-alpha.694fd6.0<br/>@codecademy/markdown-overrides@0.5.64-alpha.694fd6.0<br/>@codecademy/styleguide@43.6.2-alpha.694fd6.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6116dfbab74a481442ebdf09--gamut-preview.netlify.app>https://6116dfbab74a481442ebdf09--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6116dfbab74a481442ebdf09>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.8.4-alpha.514bb0.0<br/>@codecademy/gamut-illustrations@0.17.18-alpha.514bb0.0<br/>@codecademy/gamut-kit@0.5.32-alpha.514bb0.0<br/>@codecademy/gamut-labs@19.20.7-alpha.514bb0.0<br/>@codecademy/gamut-patterns@0.5.6-alpha.514bb0.0<br/>@codecademy/gamut-styles@11.8.1-alpha.514bb0.0<br/>@codecademy/gamut-tests@2.4.21-alpha.514bb0.0<br/>@codecademy/gamut@36.4.2-alpha.514bb0.0<br/>@codecademy/konamimojisplosion@0.2.52-alpha.514bb0.0<br/>@codecademy/markdown-overrides@0.5.64-alpha.514bb0.0<br/>@codecademy/styleguide@43.6.2-alpha.514bb0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://611a77701a67f613e051c4dc--gamut-preview.netlify.app>https://611a77701a67f613e051c4dc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/611a77701a67f613e051c4dc>Deploy Logs</a></p>

**Pull request merged**